### ISSUE TITLE
readme file example doesn't  have  { valueEncoding: 'json' }

### ISSUE THREAD

**Issue by hems** [eugeneware/jsonquery-engine#4]
which i only found on the test file: https://github.com/eugeneware/jsonquery-engine/blob/master/test/level-plan.js#L28

**Comment by eugeneware** [comment]
Thanks. Fixed in aa01bbcb63c5f6801ef936a52120356c414c5979

**Comment by hems** [comment]
yeah actually the README file could be a bit more friendly, like the "makeSomeData" method returns a specific format, which isn't a standard array of objects and so on.

i know you guys know what you doing, so you probably don't mind that most users won't understand and will stay away from the project.
